pkg/virt-operator/resource/generate/components/daemonsets.go

```diff
@@ -110,6 +110,30 @@ func NewHandlerDaemonSet(namespace, repository, imagePrefix, version, launcherVe
 		}
 	}
 
+	// If there is any image pull secret added to the `virt-handler` deployment
+	// it can mean that `virt-handler` is using private image. Therefore, we must
+	// add `virt-launcher` container that will pre-pull and keep the (probably)
+	// custom image of `virt-launcher`.
+	// Note that we cannot make it an init container because the `virt-launcher`
+	// image could be garbage collected by the kubelet.
+	// Note that we cannot add `imagePullSecrets` to `virt-launcher` as this could
+	// be a security risk - user could use this secret and abuse it.
+	if len(imagePullSecrets) > 0 {
+		pod.Containers = append(pod.Containers, corev1.Container{
+			Name:            "virt-launcher-image-holder",
+			Image:           launcherImage,
+			ImagePullPolicy: corev1.PullIfNotPresent,
+			Command:         []string{"/bin/sh", "-c"},
+			Args:            []string{"sleep infinity"},
+			Resources: corev1.ResourceRequirements{
+				Limits: map[corev1.ResourceName]resource.Quantity{
+					corev1.ResourceCPU:    resource.MustParse("50m"),
```

Comment:
Can you increase that to `100m`? I know out of first hand that `containerd` has issues with values slower than that  which can lead to startup delays of up to a minute.